packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -22,12 +22,11 @@ import { FormattedMessage } from "react-intl";
 import { createComponentSlot } from "../helpers/createComponentSlot";
 import { ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
 
-export type CrudMoreActionsMenuClassKey = "root" | "group" | "divider" | "button" | "chip" | "menuItem";
+export type CrudMoreActionsMenuClassKey = "root" | "group" | "button" | "chip";
```

Comment:
Why remove these slots? 
This is a breaking change that removes the ability to override styles/props.

Reply:
I mean the slots themselves (components created with `createComponentSlot`), not just the values in the class-key type